Corona-Warn-App/src/main/res/layout/fragment_interoperability_configuration.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <data>
+        <variable
+            name="interopViewModel"
+            type="de.rki.coronawarnapp.ui.viewmodel.InteroperabilityViewModel" />
+    </data>
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <include
+            android:id="@+id/interop_config_header"
```

Comment:
We write out everything in an id, e.g. `interoperability_configuation_header`

Reply:
Added it as To Dos in the description of the PR. Also will adjust the Ids